### DISCUSSION TITLE
Does --accept-package-agreements only work with msstore packages?

### DISCUSSION THREAD

**Discussion by rkeithhill-keysight** [microsoft/winget-pkgs#58064] [Q&A] [answered]
I am testing a package but it installs (silentWithProgress) without me having to accept any sort of agreement. It is an `exe` installer (InstallBuilder).  

I see there is an `Agreements` field in the schema but I don't see any packages that actually use that field.  How do I get winget to prompt the user to accept the license agreement for any sort of silent install?  Note: the source for this package will be winget and not msstore.

**Comment by florelis** [comment, marked as answer]
`--accept-package-agreements` works with packages from any source, if the packages from the source have agreements. This repository has the restriction that only the verified developer for a package can add `Agreements` to it, however the business process to become a verified developer is still being worked out.

See #27937, #100
